Var/Makefile

```diff
@@ -0,0 +1,14 @@
+CC = gcc
+CFLAGS = -Wall -Werror -g
+BIN = main
+OBJS = *.o
+	   
+all: $(OBJS)
+	@$(CC) $(CFLAGS) $(OBJS) -o $(BIN)
+	@rm *.o
+
+%.o: src/%.c
+	@$(CC) $(CFLAGS) -c $^
+	
+clean:
+	-@rm $(BIN) *.o $(BIN).exe 2>/dev/null || true
```

Comment:
If the Makefile is running on Linux, why include `main.exe` here?
Also, `2>/dev/null || true` can be replaced with `rm -f` instead.

Reply:
Good point. Because im using both wsl and git bash (on windows) so im using both windows' and linux' gcc